fence/models.py

```diff
@@ -592,6 +592,40 @@ class UpstreamRefreshToken(Base):
     expires = Column(BigInteger, nullable=False)
 
 
+class IdPToUser(Base):
+    # IdP & IdP sub mapping to Gen3 User sub
+
+    __tablename__ = "idp_to_user"
+
+    sub = Column(String(), primary_key=True)
+
+    fk_to_idp = Column(
+        Integer, ForeignKey(IdentityProvider.id, ondelete="CASCADE"), nullable=False
```

Comment:
in order to make a multi-column primary key (idp+sub) I think you need this:
```suggestion
        Integer, ForeignKey(IdentityProvider.id, ondelete="CASCADE"), nullable=False, primary_key=True
```

Reply:
you should recreate the table and ensure that's what happens, but I think that's how it should work